public/challenge-contributions/5/gdespolov.html

```diff
@@ -1,11 +1,103 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Challenge 5</title>
-</head>
-<body>
-  <h1>This is the template HTML file for this challenge.</h1>
-</body>
-</html>
\ No newline at end of file
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Challange 5 - Horizontal Scroll Menu</title>
+    <style>
+      body {
+        font-family: Arial, sans-serif;
+        overflow-x: hidden;
+      }
+
+      .menu-bar {
+        position: fixed;
+        bottom: 0;
+        left: 0;
+        right: 0;
+        background-color: #e10600;
+        white-space: nowrap;
+        padding: 10px;
+      }
+
+      .menu-bar__wrapper {
+        box-sizing: border-box;
+        max-width: 90ch;
+        margin: 0 auto;
+        display: flex;
+        overflow-x: auto;
+        padding: 0 0 10px 0;
+      }
+
+      .menu-bar__wrapper::-webkit-scrollbar {
+        height: 8px;
+      }
+
+      .menu-bar__wrapper::-webkit-scrollbar-thumb {
+        background-color: #ffffff;
+        border-radius: 8px;
+      }
+
+      .menu-bar__item {
+        display: inline-block;
+        padding: 15px 20px;
+        margin: 0 5px;
+        color: white;
+        text-decoration: none;
+        flex-shrink: 0;
+        border-bottom-right-radius: 8px;
+        border-top-left-radius: 8px;
+      }
+
+      .menu-bar__item--active {
+        font-weight: 800;
+        border-right: 1px solid white;
+        border-bottom: 1px solid white;
+        border-bottom-right-radius: 8px;
+        border-top-left-radius: 8px;
+      }
+
+      .menu-bar__item:hover {
+        background-color: none;
```

Comment:
This is not valid value for background-color. Please fix it.
![image](https://github.com/rumenpetrov/fe-hunger-games/assets/7723928/80a3afa3-e6c5-488d-bebf-4beafbd1471f)